tools/rosbash/rosbash

```diff
@@ -1026,4 +1082,6 @@ complete -F "_roscomplete_rossrv" "rossrv"
 complete -F "_roscomplete_roscreate_pkg" "roscreate-pkg"
 complete -F "_roscomplete_roswtf" -o filenames "roswtf"
 complete -F "_roscomplete_rosconsole" "rosconsole"
-complete -F "_roscomplete_file" "rosmv"
+complete -F "_roscomplete_filemv" "rosmv"
+
+
```

Comment:
Please remove added extra empty lines at the end of the file.

Also the file node bits have been changed from 644 to 755. Please revert too.

Reply:
Changes done to all the files